src/Tizen.TV.UIControls.Forms/Focus.cs

```diff
@@ -182,27 +182,27 @@ static void KeyHandler(RemoteControlKeyEventArgs args)
         }
     }
 
-    class PlatformFocusUpEffect : RoutingEffect
+    public class FocusUpEffect : RoutingEffect
     {
-        public PlatformFocusUpEffect() : base("TizenTVUIControl.PlatformFocusUpEffect")
+        public FocusUpEffect() : base("TizenTVUIControl.PlatformFocusUpEffect")
```

Comment:
Doesn't we need to change `effectID` as well?

Reply:
I've updated effectId